engine/runtime/src/main/java/org/enso/interpreter/runtime/data/FunctionThrowingIOException.java

```diff
@@ -0,0 +1,8 @@
+package org.enso.interpreter.runtime.data;
+
+import java.io.IOException;
+
+@FunctionalInterface
+public interface FunctionThrowingIOException<T, R> {
```

Comment:
Java is such a great language

Reply:
Turned out I didn't actually need it, but yeah, it is great...